### ISSUE TITLE
Tinkering skill Time-Efficiency appears to be incorrect.

### ISSUE THREAD

**Issue by ArcaneMusic** [TBartl/space-station-13-idle#285]
Howdy!
On the live version of the game, with the first tinking efficiency trait, you're intended to get 25% efficiency on production towards crafted items built consecutively, with the first tier capping at 25%. However, This was the result upon testing:
![image](https://user-images.githubusercontent.com/41715314/87458552-286c1080-c5d8-11ea-95cf-411d17f27101.png)
To
![image](https://user-images.githubusercontent.com/41715314/87458500-168a6d80-c5d8-11ea-9a28-f986e026eaff.png)
![image](https://user-images.githubusercontent.com/41715314/87459257-29ea0880-c5d9-11ea-81cd-f0c329e2db24.png)

It should be being reduced down to 7.5 seconds if the 25% is accurate. I also don't have enough time in-game yet to know how far this trait goes, so feel free to tell me if this is intended.

**Comment by remona-minett** [comment]
Hey, so I know this issue is very old but I'm getting back to you to tell you this is the way the math in the game works.
What the game does is basically:
`(time to craft) / (bonus * 100)`.
In simpler terms, take 10 and divide it by 1.25, and you receive 8. 
![image](https://user-images.githubusercontent.com/50187357/89770891-454f1300-dab4-11ea-91bd-7f6835eac158.png)

It's very odd, but it is *correct*, in some form or another. Thanks for bringing this up though.

**Comment by ArcaneMusic** [comment]
Sounds good, just wanted to check.
